.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       clang_format: true
       clang_format_ignore: |
         src/eckit/contrib
+        third-party
```

Comment:
Why are we creating a second directory here. I don't see libfmt as a dependency being conceptually meaningfully different to xxHash. They are both third party contribs. Please put in the 'original' location.

Reply:
well, it is a third-party library ... same for xxHash. The naming `contribution` is just not appropriate for these because they are no contributions from externals provided just for this purpose.

Also note that in https://github.com/ecmwf/eckit/pull/187  it is to discussed to move them to `third_party` directory as well, and also use git submodules properly...

However, we want to keep the discussion with the git submodules separate. That's why this library is still vendored.

Reply:
Semantics... contrib is the directory for external contribution; personally I never heard of the term "vendored" before in this context either.
If instead this is agreed to go to a new directory third-party, then I propose to also move xxHash to this third-party to be consistent at least.